sys-boot/plymouth/plymouth-24.004.60.ebuild

```diff
@@ -0,0 +1,120 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+EAPI=8
+
+inherit meson readme.gentoo-r1 flag-o-matic
+
+SRC_URI="https://dev.gentoo.org/~aidecoe/distfiles/${CATEGORY}/${PN}/gentoo-logo.png"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://gitlab.freedesktop.org/plymouth/plymouth"
+else
+	SRC_URI="${SRC_URI} https://www.freedesktop.org/software/plymouth/releases/${P}.tar.xz"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~loong ~ppc ~ppc64 ~riscv ~sparc ~x86"
+fi
+
+DESCRIPTION="Graphical boot animation (splash) and logger"
+HOMEPAGE="https://cgit.freedesktop.org/plymouth/"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="debug +drm +gtk +pango selinux freetype +split-usr static-libs +udev doc upstart-monitoring systemd"
+
+DEPEND="
+	virtual/libc
+	>=media-libs/libpng-1.2.16:=
+	drm? ( x11-libs/libdrm )
+	gtk? (
+		dev-libs/glib:2
+		>=x11-libs/gtk+-3.14:3
+		x11-libs/cairo
+	)
+	systemd? ( sys-apps/systemd )
+	pango? (
+		>=x11-libs/pango-1.21
+		x11-libs/cairo
+	)
+	freetype? ( media-libs/freetype:2 )
+	udev? ( virtual/libudev )
+	dev-libs/libevdev
+	x11-libs/libxkbcommon
+	x11-misc/xkeyboard-config
+	upstart-monitoring? (
+		sys-apps/dbus
+		sys-libs/ncurses
+	)
+
+	elibc_musl? ( sys-libs/rpmatch-standalone )
+	app-text/docbook-xsl-stylesheets
+	dev-libs/libxslt
+	virtual/pkgconfig
+"
+BDEPEND="${DEPEND}
```

Comment:
Not sure about this, but this looks strange. Don't think all `${DEPEND}` packages are required for `${BDEPEND}`?

Reply:
yeah I don't think they are needed, I'll setup the ebuildtester to try my latest commit, if it builds I'll resolve this.